### ISSUE TITLE
i think i was haked by javascript of this website http://ethchange.online/

### ISSUE THREAD

**Issue by azizyano** [MetaMask/eth-phishing-detect#2253]
i lose my eth last day 
i viste this website leaving metamask open. do you think this website can do that?

**Comment by 409H** [comment]
Do you remember confirming a popup message from MetaMask?

**Comment by azizyano** [comment]
no. maybe i was hacked before visiting this website but the time of hack attack the suspect website is this one.
